libs/auth/scopes/src/lib/mms.scope.ts

```diff
@@ -0,0 +1,3 @@
+export enum MmsScope {
```

Comment:
Shouldn't this be in capital letters?
```suggestion
export enum MMSScope {
```

Reply:
Fixed